src/main/java/io/strimzi/kafka/bridge/BridgeConsumerEndpoint.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 Red Hat Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.strimzi.kafka.bridge;
+
+/**
+ * Interface for Sink Endpoints contains consumer specific methods
+ */
+public interface BridgeConsumerEndpoint {
+
+    void createConsumer(Endpoint<?> endpoint);
+
+    void subscribeToTopic(Endpoint<?> endpoint);
+
+    void consume(Endpoint<?> endpoint);
```

Comment:
In any case we don't need a poll/consume method in an interface because it's not going to be called from outside. If we use the `sink.handle` for handling HTTP client requests (as suggested in the following comments), it will just be a one more protected method in the base sink class for calling a poll on Vert.x consumer.